scripts/get_client.py

```diff
@@ -0,0 +1,17 @@
+from python_freeipa import ClientMeta
+import yaml
+
+
+def get_client() -> ClientMeta:
+    with open("freeipa_config.yml", "r") as stream:
+        try:
+            freeipa_config = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```

Comment:
Don't try and catch this exception. At this point there's nothing more we can do so just let it throw all the way up